pages/workspace/workspace.ejs

```diff
@@ -16,24 +16,51 @@
           <button type="button" class="close" data-dismiss="modal" aria-label="Close">
             <span aria-hidden="true">&times;</span>
           </button>
-        </div>
+        </div><!-- .modal-header -->
         <div class="modal-body">
           <p>Are you sure you want to reboot the virtual machine?<p>
           <ul class="fa-ul">
-            <li><span class="fa-li"><i class="fas fa-window-restore" aria-hidden="true"></i></span>Your workspace will reboot into this same tab.</li>
             <li><span class="fa-li"><i class="fas fa-save" aria-hidden="true"></i></span>Your files will remain intact through the reboot.</li>
+            <li><span class="fa-li"><i class="fas fa-window-restore" aria-hidden="true"></i></span>Your workspace will reboot into this same tab.</li>
           </ul>
-        </div>
+        </div><!-- .modal-body -->
         <div class="modal-footer">
           <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-          <a class="btn btn-info" href="/pl/workspace/<%= workspace_id %>/reboot">
+          <a class="btn btn-primary" href="/pl/workspace/<%= workspace_id %>/reboot">
             <i class="fas fa-sync text-white" aria-hidden="true"></i>
             Reboot
           </a>
-        </div>
+        </div><!-- .modal-footer -->
       </div>
     </div>
-  </div>
+  </div><!-- #rebootModal -->
+
+  <div class="modal fade" id="resetModal" tabindex="-1" aria-labelledby="resetModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h1 class="h5 modal-title" id="resetModalLabel">Reset the workspace</h1>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div><!-- .modal-header -->
+        <div class="modal-body">
+          <p>Are you sure you want to reset the virtual machine?<p>
+          <ul class="fa-ul">
+            <li><span class="fa-li"><i class="fas fa-exclamation-triangle" aria-hidden="true"></i></span>Your file changes <em>will be lost</em>. The workspace will return to its original state upon reset.</li>
```

Comment:
Can we make this more noticeable? Maybe make the fa-exclamation-triangle red? Or the whole sentence above red? Or something else?

Reply:
Tweaked to this:
![image](https://user-images.githubusercontent.com/2503508/90590711-ddc94f80-e1a6-11ea-96a5-cfb94703ffb5.png)

Reply:
I love it! This seems impossible to overlook (although I'm sure some student will manage it 😄).